modules/apps/29-fee/types/tx.pb.go

```diff
@@ -0,0 +1,1535 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: ibc/applications/fee/v1/tx.proto
+
+package types
+
+import (
+	context "context"
+	fmt "fmt"
+	types "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	_ "github.com/gogo/protobuf/gogoproto"
+	grpc1 "github.com/gogo/protobuf/grpc"
+	proto "github.com/gogo/protobuf/proto"
+	grpc "google.golang.org/grpc"
+	codes "google.golang.org/grpc/codes"
+	status "google.golang.org/grpc/status"
+	io "io"
+	math "math"
+	math_bits "math/bits"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
+
+// MsgRegisterCounterpartyAddress is the request type for registering the counterparty address
+type MsgRegisterCounterpartyAddress struct {
+	Address             string `protobuf:"bytes,1,opt,name=address,proto3" json:"address,omitempty"`
+	CounterpartyAddress string `protobuf:"bytes,2,opt,name=counterparty_address,json=counterpartyAddress,proto3" json:"counterparty_address,omitempty" yaml:"counterparty_address"`
+	ChannelId           string `protobuf:"bytes,3,opt,name=channel_id,json=channelId,proto3" json:"channel_id,omitempty" yaml:"channel_id"`
```

Comment:
We just realized that with interchain accounts, this may be cumbersome for relayers to have to register the counterparty address for each new channel added. Would it make more sense if we register counterparty address by client ID or connection ID?

Reply:
Issue [#1133](https://github.com/cosmos/ibc-go/issues/1133) opened to discuss.